.taskcluster.yml

```diff
@@ -53,6 +56,35 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("test_mozilla_central")}
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '1 hour'}
+        provisionerId: proj-relman
+        workerType: ci
+        payload:
+          maxRunTime: 3600
+          image: "rust:buster"
+          command:
+            - "/bin/bash"
+            - "-cx"
+            - "git clone --recursive --quiet ${repository} &&
+               git clone --depth 1 --quiet https://github.com/mozilla/${mozilla_central}.git &&
```

Comment:
This command should fail when the repo is already in the cache (`fatal: il percorso di destinazione 'rust-code-analysis' esiste già e non è una directory vuota.`).

Reply:
How can we avoid that?